source/tests/integration/end-to-end/code/literal_source/RegexSource.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package com.facebook.marianatrench.integrationtests;
+
+import java.text.MessageFormat;
+
+public class RegexSource {
+
+  static String getAttackerControlledInput() {
+    return "";
+  }
+
+  static void sink(String query, String aci) {
+  }
+
+  // -------------------------------------------------------
+  //  Issues
+  // -------------------------------------------------------
+  public static void testRegexSource() {
```

Comment:
nit: we usually have the word "issue" somewhere in the method name if we're expecting an issue there... so maybe `testRegexSourceIssue()` or something? (and `testRegexSourceNoMatchNoIssue()` when there are no issues expected)